src/utilities/collection-utils.test.ts

```diff
@@ -16,7 +16,7 @@ describe("CollectionUtils", () => {
             const result = CollectionUtils.hasValues(collection);
 
             // Assert
-            expect(result).toBeTrue();
+            expect(result).toBe(true);
```

Comment:
Question: Why the change from the `toBeTrue` matcher?

Reply:
If I had to guess, it's probably an IDE thing - I believe we still need to update `setupTests.ts` to have an ES6-style import of `jest-extended` for the TS server to properly recognize them in an editor like VS Code. However, I would suggest changing them back, @dtnhinh. We can open an issue to address the `jest-extended` import